index.js

```diff
@@ -18,7 +18,18 @@ async function start({
   });
   presenceWss.on('connection', (s, req) => {
     if (!initialized) {
-      engine.initVr();
+      const resultCode = engine.initVr();
+
+      // Don't continue if there was an error initializing OpenVR
+      // See https://github.com/ValveSoftware/openvr/wiki/HmdError for error codes
+      if (resultCode === 100) {
+        console.error('OpenVR Installation not found', resultCode);
+        return process.exit(resultCode);
```

Comment:
I don't think this will do what was intended.

It will presumably return the result of `process.exit` to the handler of `presenceWss.on`. I'm not sure why that would be useful -- it will probably just be thrown out in that handler.

The intended thing was probably to return it as the result of some promise in this closure, which would need a different strategy.

Reply:
Hmm it did seem to work when I test it, mainly because I think the `process.exit` quit the entire node process immediately with the `resultCode` status code. The `return` in that line is probably unnecessary though because the exit will stop any more code from running anyway.

But I think you're suggesting wrapping the whole `start` function in a promise and just calling `reject(resultCode)` instead of the `return process.exit(resultCode)` which I could do instead?

Reply:
> But I think you're suggesting wrapping the whole start function in a promise and just calling reject(resultCode) instead of the return process.exit(resultCode) which I could do instead?

Yes, that might work.